accepted/PSR-1-basic-coding-standard.md

```diff
@@ -97,14 +97,14 @@ if (! function_exists('bar')) {
 }
 ~~~
 
-## 3. Namespace and Class Names
+## 3. Namespace, Class Names and Enums
```

Comment:
I think the only needed change is in section 4.:
Changing
```
The term "class" refers to all classes, interfaces, and traits.
```
to
```
The term "class" refers to all classes, interfaces, enums and traits.
```
Or something similar.

Reply:
But we can also consider that enum is just a special type of class. So no changes would be needed.